package.json

```diff
@@ -197,6 +197,7 @@
     "prettier-bytes": "^1.0.4",
     "qr-image": "^3.2.0",
     "react-json-view-lite": "^0.9.5",
+    "react-markdown": "^9.0.1",
```

Comment:
nit: should we pin an exact version here?

Reply:
good catch, yes.